editor/filesystem_dock.cpp

```diff
@@ -3767,6 +3773,8 @@ FileSystemDock::FileSystemDock() {
 
 	split_box = memnew(SplitContainer);
 	split_box->set_v_size_flags(SIZE_EXPAND_FILL);
+	split_box->connect("dragged", callable_mp(this, &FileSystemDock::_split_dragged));
+	split_box_offset_h = 240 * EDSCALE;
```

Comment:
Isn't that too big for a default value? I'd use 120.

Reply:
On the default layout they both look fine, but when the left panel is expanded, the folder tree stays shrinked when offset is small.
| offset = 120  | offset = 240 |
| ------------- | ------------- |
| ![offset120](https://github.com/godotengine/godot/assets/56537502/ac3a980f-2b84-4492-b15d-b1d15b2d4013) | ![offset240](https://github.com/godotengine/godot/assets/56537502/894d5442-c6bb-41d3-8eda-307bb69654f5) |

Reply:
Eh, maybe it's fine then.